### DISCUSSION TITLE
mobile phone client got Hydration error when accessing official website through Baidu search, how to fix it?

### DISCUSSION THREAD

**Discussion by bsg1981** [nuxt/nuxt#17343] [Questions]
"nuxt3 version": "3.0.0-27356801.e9128f3"

Steps to reproduce
1.through Baidu search by keyword "数环通" got url as below
   a. open mobile phone client browser(Google Chrome)
   b. visit www.baidu.com
   c. input key words "数环通"
   d. search by key words "数环通"
   e. click the first  item form search results
       this first  item's URL is : https://www.solinkup.com/#1?bd_vid=10734887471542296795

2.click this url and press key F12 got err as below 
err info: Hydration completed but contains mismatches.

how to fix this issue?

**Comment by Flanker72** [comment]
Seems that `bd_vid` query param causes this problem: trying to access the link without it in the URL results in the correct behavior. There is no code to check, so I can't help with the solution: it's not clear what your application does and how it handles the query params.
